spacemacs/keybindings.el

```diff
@@ -413,3 +413,10 @@ Ensure that helm is required before calling FUNC."
   ("w" other-window                          :doc (spacemacs//window-manipulation-move-doc)))
 
 ;; end of Window Manipulation Micro State
+
+;; linum margin bindings ---------------------------------------------------------------
+
+(global-set-key (kbd "<left-margin> <down-mouse-1>") 'spacemacs/md-select-linum)
+(global-set-key (kbd "<left-margin> <mouse-1>") 'spacemacs/mu-select-linum)
+(global-set-key (kbd "<left-margin> <double-mouse-1>") 'spacemacs/select-current-block)
+(global-set-key (kbd "<left-margin> <drag-mouse-1>") 'spacemacs/mu-select-linum)
```

Comment:
Move this block of global bindings at the top of the file before the leader section.